spacy/lang/char_classes.py

```diff
@@ -213,8 +213,12 @@
 _ukrainian_upper = r"А-ЩЮЯІЇЄҐ"
 _ukrainian = r"а-щюяіїєґА-ЩЮЯІЇЄҐ"
 
-_upper = LATIN_UPPER + _russian_upper + _tatar_upper + _greek_upper + _ukrainian_upper
-_lower = LATIN_LOWER + _russian_lower + _tatar_lower + _greek_lower + _ukrainian_lower
+_macedonian_lower = r"ѓѕјњќѐѝа-ик-ш"
+_macedonian_upper = r"ЃЅЈЊЌЀЍА-ИК-Ш"
+_macedonian = r"ѓѕјњќѐѝа-ик-шЃЅЈЊЌЀЍА-ИК-Ш"
```

Comment:
How much does this overlap with the Russian and Ukrainian character ranges? Do you think it makes sense to consolidate Russian / Tatar / Ukrainian / Macedonian here?

Reply:
24 characters are overlapping with both Russian and Ukranian (\u0430 - \u0448 excluding \u0439). If we don't want character redundancy, I will remove them from the Macedonian set. That way, there will be 8 new characters left to be added here.